docs/store.js

```diff
@@ -1,26 +1,14 @@
 // Firebase storage manager.
 var firebaseCalendarStorage = function(params) {
```

Comment:
If this is a class created with `new` then it should start with a capital letter.